src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -2400,4 +2406,23 @@ export default class GeFormRenderer extends LightningElement{
                 [this.batchCurrencyIsoCode]);
         }
     }
+
+    isFieldUsedForCardholderName(fieldMapping) {
+        const cardholderNameFields = [CONTACT_LAST_NAME_INFO, CONTACT_FIRST_NAME_INFO, ACCOUNT_NAME_FIELD];
+        return cardholderNameFields.find( f => apiNameFor(f) === fieldMapping.Target_Field_API_Name);
+    }
+
+    updateCardholderState(valueObjectFromRecord, fieldMapping) {
+        const value = this.getFieldValueForFormState(valueObjectFromRecord, fieldMapping);
+        this.cardholderNamesNotInTemplate[fieldMapping.Source_Field_API_Name] = value;
+    }
+
+    getFieldValueFromCardholderState(fieldApiNameOrFieldReference) {
```

Comment:
The first four lines of the function are duplicated with the first four lines in `getFieldValueFromFormState`. I think that's an opportunity for refactor or re-use. I'm also not sure we need to check for `fieldApiName` or `fieldReference` here, this function is called once and at that point, we know we know what we're passing to it (a field reference).